### ISSUE TITLE
Use consistent capitalization for hash URIs

### ISSUE THREAD

**Issue by peterdesmet** [oSoc15/ipt-dcat#52]
Current use:

```
/#catalog
/#Organization
/#dataset
```

I notice the Datatank [is mixing them too](http://demo.thedatatank.com/api/dcat) (including trailing slash):

```
/#organization
#Government
#Mobility
```

@pietercolpaert, is there is a convention? I'd propose:

```
/#Catalog
/#Organization
/#Dataset
```

**Comment by simon-vc** [comment]
Pieter said its better without the slash
So I'll go with
#Catalog
#Organization
#Dataset

**Comment by simon-vc** [comment]
There is no convention at all, the choice is free

**Comment by peterdesmet** [comment]
Confirmed fixed in `v1.2`
